src/EnsembleKalmanProcess.jl

```diff
@@ -163,14 +160,16 @@ function EnsembleKalmanProcess(
     P <: Process,
     FM <: FailureHandlingMethod,
     LM <: LocalizationMethod,
+    OS <: ObservationSeries,
 }
 
     #initial parameters stored as columns
     init_params = DataContainer(params, data_are_columns = true)
 
     # dimensionality
     N_par, N_ens = size(init_params) #stored with data as columns
-    N_obs = length(obs_mean)
+    obs = get_obs(observation_series)
```

Comment:
Could be helpful to have a quick comment here or change the variable name `N_obs` to indicated whether it's number of obs in a batch or total number of obs

Reply:
Done, and added a note to the glossary.md

I think in future we will want to remove this exact line, as currently this pins the Localizer (the only place N_obs is used here) to work with a fixed minibatch size